tests/CliTest.php

```diff
@@ -13,7 +13,7 @@ class CliTest extends TestCase
      */
     public function test deploying requires mandatory files to exist()
     {
-        $process = new Process(__DIR__ . '/../bref deploy', __DIR__ . '/Fixture/EmptyDirectory');
+        $process = new Process([__DIR__ . '/../bref deploy', __DIR__ . '/Fixture/EmptyDirectory']);
```

Comment:
you guessed it... ;)

```suggestion
        $process = new Process([__DIR__ . '/../bref deploy'], __DIR__ . '/Fixture/EmptyDirectory');
```